config/tests/testgrids/config_test.go

```diff
@@ -351,6 +351,7 @@ var noPresubmitsInTestgridPrefixes = []string{
 	"GoogleCloudPlatform/k8s-multicluster-ingress",
 	"kubeflow/pipelines",
 	"kubernetes-sigs/cluster-capacity",
+	"kubernetes-sigs/descheduler",
```

Comment:
Why no testgrid? We are trying to whittle this list down

Reply:
I copy-pasted `kubernetes-sigs/scheduler-plugins`. I have no objections to put it some place else. What's the better place to put it in?

Reply:
https://testgrid.k8s.io/sig-scheduling